### PR TITLE
Move the most important links to a better place

### PR DESCRIPTION
**Pull request by jan-cerny**
This increases visibility of the stuff that we are doing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -261,15 +266,6 @@ decided to change the name of the project to `ComplianceAsCode`, in order to avo
 We envision that the future will be format-agnostic. That's why opted for an
 abstraction instead of using XCCDF for the input format.
 
-## Further reading
-
-The SSG homepage is [https://www.open-scap.org/security-policies/scap-security-guide/](https://www.open-scap.org/security-policies/scap-security-guide/).
```
Comment (ggbecker) [inline README.md:266]: I believe keeping the link to the web page is still relevant. I think you can transform this section into a web site dedicated section for example.

Comment (jan-cerny) [reply README.md:266]: That web page is very outdated, it doesn't even mention the rename that our project did long time ago, frequently mentions RHEL 6 and it doesn't add any useful information on top of readthedocs or this readme.

**Review by Mab879 — APPROVED** [review]

**Comment by Mab879** [comment]
I approve, I will leave this open for day to get more feedback.

File: README.md (hunk 1 above)
Comment (Mab879) [reply README.md:266]: I agree that Open-SCAP.org page is very outdated, leaving it out is not too big of an issue.

**Pull request merged**